### ISSUE TITLE
Verification of results for structural adapters

### ISSUE THREAD

**Issue by peterjamesnugent** [BHoM/ETABS_Toolkit#377]
#### Definition of the test :

I've produced a script to create two (for now) simple models:
- A simply supported beam,
- A free standing column

A number of different loads are applied:
- Bar uniformly distributed load
- Shear Y and Shear Z applied at mid span
- Torsion applied at free end
- Axial force applied at free end

For this toolkit can you please run the script to create the model, analyse the model and verify that the results are pulled.

This is also a good opportunity to ensure the results follow the [BHoM's Structural Conventions](https://github.com/BHoM/documentation/wiki/BHoM-Structural-Conventions). 

Please save your model in an appropriate folder in [this location](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/16_Test%20Data/Structural%20Models%20for%20Results%20Verificaiton/Models?csf=1&web=1&e=kMCbj7).

The script to push the generic models[ can be found here](https://burohappold.sharepoint.com/:f:/r/sites/BHoM/02_Current/16_Test%20Data/Structural%20Models%20for%20Results%20Verificaiton?csf=1&web=1&e=mGevfI).

Also it should be noted that you will be required to reinitialise the `AdapterId` components (I've highlighted them in red) - see here for the issue: https://github.com/BHoM/BHoM_Adapter/issues/283.

**Comment by peterjamesnugent** [comment]
It also worth noting that the models I have created are in SI Units - so unless this toolkit implements the Units_Engine it should also be in SI Units to allow easy comparison of results between calculated and expected values.

**Comment by IsakLarbornBH** [comment]
I've created and uploaded 3 models using the script now, for each supported version of ETABS. Results are pulling out fine and are identical between the three versions. 

ETABS2016
<img width="531" alt="ETABS2016" src="https://user-images.githubusercontent.com/72074998/106593615-efca5f00-6548-11eb-842b-64fa3c947ed5.png">
ETABS17
<img width="611" alt="ETABS17" src="https://user-images.githubusercontent.com/72074998/106593630-f48f1300-6548-11eb-98d6-93ad8679b3da.png">
ETABS18
<img width="505" alt="ETABS18" src="https://user-images.githubusercontent.com/72074998/106593636-f6f16d00-6548-11eb-874c-5e41927f18b0.png">

Since the script didn't have ETABS18 support, I added that myself, and I've uploaded that version of the script as well next to the original one.

Hopefully that's all good then :)
